systems-manager.tf

```diff
@@ -0,0 +1,96 @@
+# AWS KMS alias used for encryption/decryption of SSM secure strings
+variable "kms_alias_name_ssm" {
+  type        = string
+  default     = "alias/aws/ssm"
+  description = "KMS alias name for SSM"
+}
+
+variable "ssm_parameters_enabled" {
+  type        = bool
+  default     = false
+  description = "If `true` create SSM keys for the database user and password."
```

Comment:
This is controlling more than just the PStore params for user + pass since you're also including hostname + port. I'd update or better yet make this a more generic "for the database info".